Engine.Core/TestHost.fs

```diff
@@ -1,11 +1,13 @@
 ﻿namespace R4nd0mApps.TddStud10.Engine.Core
-
+open System.IO
+open System.Runtime.Serialization.Json
+open System.Text
 
 module TestHost =
     open R4nd0mApps.TddStud10.Common.Domain
 
     let buildCommandLine cmd (rsp : RunStartParams) =
-        sprintf "\"%s\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\""
+        sprintf "\"%s\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\""

```

Comment:
Windows command line has a length limit of 4096 unicode characters. For 10 arguments, it means 410 characters average per argument, which should be OK for now.